src/Network/Network/NetworkWatcher/ConnectionMonitor/NewAzureNetworkWatcherConnectionMonitorCommand.cs

```diff
@@ -64,48 +70,221 @@ public class NewAzureNetworkWatcherConnectionMonitorCommand : ConnectionMonitorB
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The list of test group.",
+            ParameterSetName = "SetByResource")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The list of test group.",
+            ParameterSetName = "SetByName")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The list of test group.",
+            ParameterSetName = "SetByLocation")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The list of test group.",
+            ParameterSetName = "SetByConnectionMonitorV2")]
+        [ValidateNotNullOrEmpty]
+        public List<PSNetworkWatcherConnectionMonitorTestGroupObject> TestGroup { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The connection monitor output.",
+            ParameterSetName = "SetByResource")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The connection monitor output.",
+            ParameterSetName = "SetByName")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The connection monitor output.",
+            ParameterSetName = "SetByLocation")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The connection monitor output.",
+            ParameterSetName = "SetByConnectionMonitorV2")]
+        //TODO
+        //[ValidateNotNullOrEmpty]
+        public List<PSNetworkWatcherConnectionMonitorOutputObject> Output { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Notes associated with connection monitor.",
+            ParameterSetName = "SetByResource")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Notes associated with connection monitor.",
+            ParameterSetName = "SetByName")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Notes associated with connection monitor.",
+            ParameterSetName = "SetByLocation")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Notes associated with connection monitor.",
+            ParameterSetName = "SetByConnectionMonitorV2")]
+        //TODO
+        //[ValidateNotNullOrEmpty]
```

Comment:
remove commented attribute.

Also you should not use a plural parameter like 'Notes', 'Note' should be fine